Sources/PointFree/Emails/EmailLayouts.swift

```diff
@@ -14,12 +14,15 @@ struct SimpleEmailLayoutData<A> {
   let data: A
 }
 
+let emailStylesheet = styleguide
+  <> a % key("text-decoration", "underline")
```

Comment:
we'll need to overwrite some styles from the styleguide specifically for emails, and this is the beginning of that. email clients dont support `:hover`, so gotta make sure all links are underlined...